example/src/main/java/com/stripe/example/activity/PaymentAuthActivity.kt

```diff
@@ -1,60 +1,32 @@
 package com.stripe.example.activity
 
-import android.content.Intent
 import android.os.Bundle
 import android.view.View
-import androidx.appcompat.app.AppCompatActivity
-import com.stripe.android.ApiResultCallback
+import androidx.lifecycle.Observer
 import com.stripe.android.PaymentAuthConfig
-import com.stripe.android.PaymentIntentResult
-import com.stripe.android.SetupIntentResult
 import com.stripe.android.Stripe
 import com.stripe.android.model.Address
 import com.stripe.android.model.ConfirmPaymentIntentParams
-import com.stripe.android.model.ConfirmSetupIntentParams
 import com.stripe.android.model.PaymentMethodCreateParams
-import com.stripe.example.R
 import com.stripe.example.Settings
-import com.stripe.example.StripeFactory
 import com.stripe.example.databinding.PaymentAuthActivityBinding
-import com.stripe.example.module.BackendApiFactory
-import com.stripe.example.service.BackendApi
-import io.reactivex.android.schedulers.AndroidSchedulers
-import io.reactivex.disposables.CompositeDisposable
-import io.reactivex.schedulers.Schedulers
-import okhttp3.ResponseBody
-import org.json.JSONException
-import org.json.JSONObject
-import java.io.IOException
-import java.lang.ref.WeakReference
 
 /**
  * An example of creating a PaymentIntent, then confirming it with [Stripe.confirmPayment]
  */
-class PaymentAuthActivity : AppCompatActivity() {
+class PaymentAuthActivity : PaymentIntentActivity() {
```

Comment:
should we rename `PaymentIntentActivity` to `StripeIntentActivity` since it also supports setup intents?